datasources/build.gradle

```diff
@@ -22,7 +22,11 @@ dependencies {
     implementation group: 'org.opensearch', name: 'opensearch-x-content', version: "${opensearch_version}"
     implementation group: 'org.opensearch', name: 'common-utils', version: "${opensearch_build}"
     implementation group: 'commons-io', name: 'commons-io', version: '2.8.0'
-    implementation 'com.amazonaws:aws-encryption-sdk-java:2.4.1'
+    // FIXME. upgrade aws-encryption-sdk-java once the bouncycastle dependency update to 1.78.1.
+    implementation ('com.amazonaws:aws-encryption-sdk-java:2.4.1') {
+        exclude group: 'org.bouncycastle', module: 'bcprov-ext-jdk18on'
+    }
+    implementation 'org.bouncycastle:bcprov-jdk18on:1.78.1'
```

Comment:
Should we pull version from core instead of hard coding? https://github.com/opensearch-project/OpenSearch/blob/main/buildSrc/version.properties#L57

Reply:
Do we even need to bring this dependency in? I don't see any usage of it in the repo

Reply:
Done. align with core version.
Prometheus datasource use encryption? @vamsi-amazon could u help confirm?

Reply:
yes, we are using it for storing basic credentials in datasources store.